packages/ui/src/components/TokenMedia/TokenFallback.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC, ComponentPropsWithoutRef } from 'react'
+import { Button, Flex, Text } from '../../primitives'
+import TokenMedia from './index'
+import { defaultHeaders } from '../../lib/swr'
+import { useReservoirClient } from '../../hooks'
+import { paths } from '@reservoir0x/reservoir-kit-client'
+
+type TokenFallbackProps = {
+  token: ComponentPropsWithoutRef<typeof TokenMedia>['token']
+  onRefreshClicked: () => void
+}
+
+const TokenFallback: FC<TokenFallbackProps> = ({ token, onRefreshClicked }) => {
+  const client = useReservoirClient()
+
+  return (
+    <Flex
+      justify="center"
+      align="center"
+      direction="column"
+      css={{ gap: '$2' }}
+    >
+      {token?.collection?.image && (
+        <img
+          style={{ width: 64, height: 64, objectFit: 'cover', borderRadius: 8 }}
+          src={token?.collection?.image}
+        />
+      )}
+      <Text style="body3" css={{ textAlign: 'center' }}>
+        No Content Available
+      </Text>
+      <Button
+        color="secondary"
+        onClick={() => {
+          onRefreshClicked()
+          const url = `${client?.apiBase}/tokens/refresh/v1`
```

Comment:
wonder if we should export some sort of fetch implementation from the client.  like `client.api('/tokens/refresh/v1')`

if not i think this should be pulled out into it's only library function anyway to abstract away the api key, version, getting the base path, parsing the json